### PR TITLE
Refactor address to country

### PR DESCRIPTION
**Pull request by chiralcentre**

### CODE REVIEW

**Review by logical-1985516 — APPROVED** [review]
Other than a few nits, LGTM

File: src/main/java/seedu/address/storage/JsonAdaptedPerson.java [hunk 1]
```diff
@@ -39,13 +39,13 @@ class JsonAdaptedPerson {
      */
     @JsonCreator
     public JsonAdaptedPerson(@JsonProperty("name") String name, @JsonProperty("phone") String phone,
-            @JsonProperty("email") String email, @JsonProperty("address") String address,
-                             @JsonProperty("status") String status, @JsonProperty("comment") String comment,
-                             @JsonProperty("tags") List<JsonAdaptedTag> tags) {
+            @JsonProperty("email") String email, @JsonProperty("country") String address,
+            @JsonProperty("status") String status, @JsonProperty("comment") String comment,
+            @JsonProperty("tags") List<JsonAdaptedTag> tags) {
```
[the remark this answers is not in the document]
Comment (chiralcentre) [reply src/main/java/seedu/address/storage/JsonAdaptedPerson.java:44]: fixed

File: src/test/java/seedu/address/logic/commands/CommandTestUtil.java [hunk 2]
```diff
@@ -56,7 +56,7 @@ public class CommandTestUtil {
     public static final String INVALID_NAME_DESC = " " + PREFIX_NAME + "James&"; // '&' not allowed in names
     public static final String INVALID_PHONE_DESC = " " + PREFIX_PHONE + "911a"; // 'a' not allowed in phones
     public static final String INVALID_EMAIL_DESC = " " + PREFIX_EMAIL + "bob!yahoo"; // missing '@' symbol
-    public static final String INVALID_ADDRESS_DESC = " " + PREFIX_ADDRESS; // empty string not allowed for addresses
+    public static final String INVALID_COUNTRY_DESC = " " + PREFIX_COUNTRY; // empty string not allowed for addresses
```
[the remark this answers is not in the document]
Comment (chiralcentre) [reply src/test/java/seedu/address/logic/commands/CommandTestUtil.java:59]: fixed

File: src/test/java/seedu/address/logic/parser/ParserUtilTest.java [hunk 3]
```diff
@@ -103,26 +103,26 @@ public void parsePhone_validValueWithWhitespace_returnsTrimmedPhone() throws Exc
...
 
     @Test
-    public void parseAddress_validValueWithWhitespace_returnsTrimmedAddress() throws Exception {
-        String addressWithWhitespace = WHITESPACE + VALID_ADDRESS + WHITESPACE;
-        Address expectedAddress = new Address(VALID_ADDRESS);
-        assertEquals(expectedAddress, ParserUtil.parseAddress(addressWithWhitespace));
+    public void parseCountry_validValueWithWhitespace_returnsTrimmedCountry() throws Exception {
+        String addressWithWhitespace = WHITESPACE + VALID_COUNTRY + WHITESPACE;
+        Country expectedCountry = new Country(VALID_COUNTRY);
+        assertEquals(expectedCountry, ParserUtil.parseCountry(addressWithWhitespace));
```
[the remark this answers is not in the document]
Comment (chiralcentre) [reply src/test/java/seedu/address/logic/parser/ParserUtilTest.java:125]: fixed

File: src/test/java/seedu/address/model/person/NameContainsKeywordsPredicateTest.java [hunk 4]
```diff
@@ -68,10 +68,10 @@ public void test_nameDoesNotContainKeywords_returnsFalse() {
         predicate = new NameContainsKeywordsPredicate(Arrays.asList("Carol"));
         assertFalse(predicate.test(new PersonBuilder().withName("Alice Bob").build()));
 
-        // Keywords match phone, email and address, but does not match name
-        predicate = new NameContainsKeywordsPredicate(Arrays.asList("12345", "alice@email.com", "Main", "Street"));
+        // Keywords match phone, email, but does not match name
+        predicate = new NameContainsKeywordsPredicate(Arrays.asList("12345", "alice@email.com"));
```
[the remark this answers is not in the document]
Comment (chiralcentre) [reply src/test/java/seedu/address/model/person/NameContainsKeywordsPredicateTest.java:72]: fixed

**Pull request merged**